todo/src/components/Container.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Todo from "./Todo";
+import { connect } from "react-redux";
+import { addTodo } from "../actions";
+import { toggleTodo } from "../actions";
+
+class Container extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      task: "",
+    };
+  }
```

Comment:
At this point, I think you're safe to start using the React shorthand:

```
class Container extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      todoText: ""
    };
  }
}
```

becomes...

```
class Container extends React.Component {
  state = {
      todoText: "",
  }
}
```